synapse/module_api/callbacks/__init__.py

```diff
@@ -24,8 +24,11 @@
     SpamCheckerModuleApiCallbacks,
 )
 
+from .third_party_event_rules_callbacks import ThirdPartyEventRulesModuleApiCallbacks
```

Comment:
FWIW I very much prefer absolute imports...

Reply:
This is the third time I've forgotten to change that after cherry-picking :sob: Thanks for the catch.

Reply:
Annoyingly Pycharm's auto-add import seems to really prefer relative imports. 😢